homeassistant/components/device_automation/__init__.py

```diff
@@ -37,19 +40,19 @@
         # The domain does not have device automations
         return None
 
-    if hasattr(platform, "async_get_triggers"):
-        return await platform.async_get_triggers(hass, device_id)
+    if hasattr(platform, fname):
+        return await getattr(platform, fname)(hass, device_id)
 
 
-async def async_get_device_automation_triggers(hass, device_id):
-    """List device triggers."""
+async def async_get_device_automations(hass, fname, device_id):
```

Comment:
If this function is only used in this file, let's prefix it with `_`.

Reply:
Fixed.